vignettes/LPWC.Rmd

```diff
@@ -132,6 +132,65 @@ The default is 10.
 Increasing iter increases computational time.
 This is only relevant for the low penalty because the high penalty uses a single C instead of searching over multiple C.
 
+## Interpreting lags
+
+
+```{r, echo = FALSE, fig.height = 2, fig.width=4, warning = FALSE}
+library(ggplot2)
+
+set.seed(29876)
+
+
+a <- rbind(c(rep(0, 5), 8, 0), c(rep(0, 4), 4.3, 0, 0)) + rnorm(2, 0, 0.5)
+
+dat <- data.frame(intensity = as.vector(a), time = rep(c(0, 5, 15, 30, 45, 60, 75), each = 2), genes = factor(rep(c(1, 2), 7)))
+
+a2 <- a
+a2[1, ] <- c(a2[1, 2:7], NA)
+dat2 <- data.frame(intensity = as.vector(a2), time = rep(c(0, 5, 15, 30, 45, 60, 75), each = 2), genes = factor(rep(c(1, 2), 7)))
+
+a3 <- a
+a3[2, ] <- c(NA, a3[2, 1:6])
+a3[1, ] <- c(a3[1, 2:7], NA)
+dat3 <- data.frame(intensity = as.vector(a3), time = rep(c(0, 5, 15, 30, 45, 60, 75), each = 2), genes = factor(rep(c(1, 2), 7)))
+
+
+plot1 <- ggplot(dat, aes(x= time, y = intensity, group = genes)) + geom_line(aes(color = genes), size = 1.5) +  labs(x = "Time (min)") + labs(y = "Intensity") 
+plot1
+
+row1 <- c(0, 5, 15, 30, 45, 60, 75)
+knitr::kable(t(data.frame(Gene1 = row1, Gene2 = row1)), align = 'c')
+
+
+
+plot2 <- ggplot(dat2, aes(x= time, y = intensity, group = genes)) + geom_line(aes(color = genes), size = 1.5) + 
+  labs(x = "Time (min)") + labs(y = "Intensity") 
+plot2
+
+row1 <- c(0, 5, 15, 30, 45, 60)
+row2 <- c(5, 15, 30, 45, 60, 75)
+
+knitr::kable(t(data.frame(Gene1 = row2, Gene2 = row1)), align = 'c')
+
+
+
+plot3 <- ggplot(dat3, aes(x= time, y = intensity, group = genes)) + geom_line(aes(color = genes), size = 1.5) + 
+  labs(x = "Time (min)") + labs(y = "Intensity") 
+plot3
+
+row1 <- c(0, 5, 15, 30, 45)
+row2 <- c(5, 15, 30, 45, 60)
+knitr::kable(t(data.frame(Gene1 = row2, Gene2 = row1)), align = 'c')
+
+
+
+```
+
+An example of the effects of applying different lags to genes 1 and 2. 
+The time series plot shows aligned expression vectors and the table shows the aligned timepoints.
+With no lags, $X_1 = 0$ and $X_2 = 0$, the temporal profiles of genes 1 and 2 are not aligned so the gene pair will have a low LPWC similarity score (top row).
```

Comment:
Should we remove the `X_1` and `X_2` notation?  I'll propose a change.

Reply:
sure